app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/statuses/case_status.scala.html

```diff
@@ -19,6 +19,10 @@
 @import uk.gov.hmrc.tariffclassificationfrontend.models.{CancelReason, Case, CaseStatus}
 @(c: Case, id: String, suppressLiveIndicator: Boolean = false)(implicit appConfig: AppConfig)
 
+@if(c.application.isLiveLiabilityOrder && !suppressLiveIndicator) {
```

Comment:
Presume this means that it comes first now?

Reply:
yep